pkgs/development/compilers/llvm/common/mlir/default.nix

```diff
@@ -71,6 +71,8 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   meta = llvm_meta // {
+    # Very broken since the dependencies aren't propagating at all with tablegen.
```

Comment:
"tblgen"?

Reply:
tablegen, tblgen, table gen. Idk how many times I've seen different ways for this *thing* to be written as by various people lol. I've updated it to say tblgen.